requirements/win/win_reqs.txt

```diff
@@ -0,0 +1,149 @@
+absl-py==2.1.0
```

Comment:
Avoid exact pinned dependency list like this. We're never going to update it.

Reply:
And we should not. Main problem is with one of the packages (triton) not supported on windows.
I found the replacement that can be installed on windows and there is only one version. I do not think it will update in the future so hardcoded all dependencies. 
But yes, except this triton package we can have unspecified libs